bookdown/00-intro.Rmd

```diff
@@ -10,16 +10,16 @@ Examples of this advanced functionality include hyperparameter tuning, feature s
 `r mlr_pkg("mlr3")` provides a domain-specific language for machine learning in R.
 We target both **practitioners** who want to quickly apply machine learning algorithms and **researchers** who want to implement, benchmark, and compare their new methods in a structured environment.
 The package is a complete rewrite of an earlier version of `r mlr_pkg("mlr")` that leverages many years of experience to provide a state-of-the-art system that is easy to use and extend.
-It is intended for users who have basic knowledge in machine learning and R that are interested in complex projects and advanced functionality rather than one-liners for one specific thing.
+It is intended for users who have basic knowledge in machine learning and R and who are interested in complex projects and advanced functionality rather than one-liners for one specific thing.
 
 **Why a Rewrite?**
 
 `r mlr_pkg("mlr")` was first released to CRAN in 2013, with the core design and architecture dating back much further.
 Over time, the addition of many features has led to a considerably more complex design that made it harder to build, maintain, and extend than we had hoped for.
-Further, the R ecosystem was not as far developed at the time, for example `r cran_pkg("data.table")` did not exist.
-Including such helpful packages makes `r mlr_pkg("mlr3")` easier to maintain, to use, and more performant, but also resulted in non-trivial API changes that would have been hard to implement and support in `r mlr_pkg("mlr")`.
-With hindsight, we saw that some of the design and architecture changes in `r mlr_pkg("mlr")` made it difficult to support new features, in particular with respect to pipelines.
-In 2018, we decided to start work on the reimplementation, and in July 2019 we released the first version of `r mlr_pkg("mlr3")` to CRAN.
+Furthermore, the R ecosystem as well as helpful packages such as `r cran_pkg("data.table")` have undergone major changes in the meantime.
+It would have been nearly impossible to integrate all these changes into the original design of `r mlr_pkg("mlr")`.
+Instead, we decided to start working on a reimplementation in 2018, which resulted in the first release of `r mlr_pkg("mlr3")` on CRAN .
+The new design and the integration of further developed R packages and features (**data.table**, **R6**, pipelines) makes `r mlr_pkg("mlr3")` much easier to maintain, easier to use, and more performant compared to `r mlr_pkg("mlr")`.
```

Comment:
```suggestion
The new design and the integration of newly developed R packages (**R6**, **future**) makes `r mlr_pkg("mlr3")` much easier to maintain and use, and more performant compared to `r mlr_pkg("mlr")`.
```

Reply:
how about `further and newly developed` R packages, this would then also include the **data.table** package.

Reply:
Ok. Seems like you need to commit the suggestion, I can't :)